noria-server/dataflow/src/node/mod.rs

```diff
@@ -342,32 +342,28 @@ impl Node {
 
 // reader replication
 impl Node {
-    pub fn has_replicas(&self) -> bool {
-        !self.replicas.is_empty()
+    pub fn has_readers(&self) -> bool {
+        !self.readers.is_empty()
     }
 
-    pub fn num_replicas(&self) -> usize {
-        self.replicas.len()
+    pub fn num_readers(&self) -> usize {
+        self.readers.len()
     }
 
-    pub fn get_replicas(&self) -> &[NodeIndex] {
-        &self.replicas[..]
+    pub fn get_readers(&self) -> &[NodeIndex] {
+        &self.readers[..]
     }
 
-    /// Returns replicas in round robin order each time the method is called.
-    pub fn next_replica(&mut self) -> Option<NodeIndex> {
-        if self.num_replicas() > 0 {
-            self.replica_index += 1;
-            self.replica_index %= self.num_replicas();
-            Some(*self.replicas.get(self.replica_index).unwrap())
+    /// Returns reader replicas in round robin order each time the method is called.
+    pub fn next_reader(&mut self) -> Option<NodeIndex> {
+        if self.num_readers() > 0 {
+            self.next_reader += 1;
```

Comment:
Interesting... I just realized that this means we'll give out the *first* reader *last*. Maybe that's what we want since the first reader is on the same machine as the rest of the write path, which is likely to be more loaded? I don't know.

Reply:
Either works! Certainly wasn't intentional.

Reply:
Maybe add it to the documentation for `next_reader`: "with the primary reader being returned last"?